src/components/articles/AllArticles.jsx

```diff
@@ -1,22 +1,32 @@
 import { useEffect, useState } from "react";
 import { getArticles } from "../../utils/getApi";
 import ArticleItem from "./ArticleItem";
+import "./articles.scss"
 
 export default function AllArticles() {
   const [allArticles, setAllArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true)
   useEffect(() => {
     getArticles().then(({ articles }) => {
       setAllArticles(articles);
+      setIsLoading(false)
+    })
+    .catch(error =>{
+      console.log(error)
```

Comment:
Rather than console logging the error, (which the user wont see unless they open dev tools, which they shouldn't need to), consider an `isError` pattern, to display something useful to the user: see the notes here... [https://notes.northcoders.com/courses/js-front-end/error-handling](url)